src/selectors/hearingEditor/labels.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type {LabelState} from '../../types';
+
+
+export const getAll = (state: LabelState): Array<Object> =>
+  state.all.map(frontId => state.byId[frontId]);
```

Comment:
It seems if there are more than 50 labels, only the 50 first ones are fetched from the server. Hence, the selector will return `undefined` for lots of labels and opening the manager will fail.